test/e2e/storage/vsphere/vsphere_volume_disksize.go

```diff
@@ -61,35 +59,38 @@ var _ = utils.SIGDescribe("Volume Disk Size [Feature:vsphere]", func() {
 		datastore = GetAndExpectStringEnvVar(StorageClassDatastoreName)
 	})
 
-	It("verify dynamically provisioned pv using storageclass with an invalid disk size fails", func() {
-		By("Invoking Test for invalid disk size")
+	It("verify dynamically provisioned pv has size rounded up correctly", func() {
+		By("Invoking Test disk size")
 		scParameters[Datastore] = datastore
 		scParameters[DiskFormat] = ThinDisk
 		diskSize := "1"
-		err := invokeInvalidDiskSizeTestNeg(client, namespace, scParameters, diskSize)
-		Expect(err).To(HaveOccurred())
-		errorMsg := `Failed to provision volume with StorageClass \"` + DiskSizeSCName + `\": A specified parameter was not correct`
-		if !strings.Contains(err.Error(), errorMsg) {
-			framework.ExpectNoError(err, errorMsg)
-		}
-	})
-})
+		expectedDiskSize := "1Mi"
+
+		By("Creating Storage Class")
+		storageclass, err := client.StorageV1().StorageClasses().Create(getVSphereStorageClassSpec(DiskSizeSCName, scParameters, nil))
+		framework.ExpectNoError(err)
```

Comment:
Error validation in this package is done in many different ways, example:
```
Expect(err).To(HaveOccurred())
framework.ExpectNoError(err, fmt.Sprintf("Failed to create storage class with err: %v", err))
framework.ExpectNoError(err)
```
For consistency's sake, in this patch I only used `framework.ExpectNoError(err)` .